recipe/meta.yaml

```diff
@@ -1,46 +1,48 @@
 {% set version = "1.2.1" %}
 
 package:
-    name: pykdtree
-    version: {{ version }}
+  name: pykdtree
+  version: {{ version }}
 
 source:
-    fn: pykdtree-{{ version }}.tar.gz
-    url: https://pypi.io/packages/source/p/pykdtree/pykdtree-{{ version }}.tar.gz
-    md5: 395603b8821954f1bffaf1913baf42a2
+  fn: pykdtree-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pykdtree/pykdtree-{{ version }}.tar.gz
+  sha256: 5d9dd0b4ad4818996ff2439d0366e514126655c034d73e122f1bb39c3c494b68
 
 build:
-    number: 0
-    script:
-        - export USE_OMP=1  # [linux]
-        - export USE_OMP=0  # [osx]
-        - set USE_OMP=0  # [win]
-        - python setup.py install --single-version-externally-managed --record record.txt
+  number: 0
+  skip: True  # [np==110 or py34]
```

Comment:
conda-forge is dropping Python 3.4 and NumPy 1.10 soon. I already removed them here to get a quicker build.